rootston/desktop.c

```diff
@@ -323,6 +323,14 @@ struct roots_desktop *desktop_create(struct roots_server *server,
 
 	desktop->server = server;
 	desktop->config = config;
+
+	desktop->xcursor_theme = roots_xcursor_theme_create("default");
+	if (desktop->xcursor_theme == NULL) {
+		wlr_list_free(desktop->views);
+		free(desktop);
+		return NULL;
```

Comment:
Does it make sense for this to be fatal? Could there be an embedded application that does not use xcursor themes at all? For instance, devices without a pointer (like a touch-screen kiosk) will never show a pointer.

I think I'm ok with this assumption since it simplifies the code and rootston is assumed to have desktop features, but we should always make sure xcursor is not required to be used in the library.

Reply:
Yeah, wlroots doesn't require the use of wlr_xcursor at all, we should be fine.

Reply:
Rootston isn't a kiosk-ready compositor, it's going to have a cursor (at least for now).